### PR TITLE
RD-1962: Migrate npm to the 8th version

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Changes consists of updating package-lock.json file to the newest version (which is compatible with the previous one)
In addition, the npm audit fix command has to be executed, just to clean the minor vulnerabilities 😉

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

N/A

## Documentation

N/A

### CODE REVIEW

File: backend/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1267 @@
+{
```
Comment (qooban) [inline backend/package-lock.json:1]: Actually - we don't need that file. `backend/package.json` is right now kind of artificial, to allow users to import from `backend` folder.

**Review by qooban — COMMENTED** [review]
Can you take a look at:
image

Is it something we should be worried about? I haven't seen that failure before.

**Comment by NorbertAmartus** [comment]
@qooban 
> Can you take a look at: image
> 
> Is it something we should be worried about? I haven't seen that failure before.

There vulnerabilities are referring  to the licences, used by some of the dependencies
It's hard for me to determine if we should do something about that

In addition, there is no information about a possible solution to that problem
snyk issue

File: backend/package-lock.json (hunk 1 above)
Comment (NorbertAmartus) [reply backend/package-lock.json:1]: Ou, okay 😁 

In that case, I'll remove the `package-lock.json` file 😉 
Thanks for the insight!

Comment (NorbertAmartus) [reply backend/package-lock.json]: Done 🎉

**Review by qooban — APPROVED** [review]
> There vulnerabilities are referring to the licences, used by some of the dependencies
> It's hard for me to determine if we should do something about that

OK, I double-checked and the problem was not introduced by this change. We used both packages in the same versions before, so let's merge it for now. I'm wondering why Snyk didn't report that before.

Could you send an e-mail to Alex about that (add me in CC)? It seems to me that MPL 2.0 is risky:
1. [Pros and Cons of using MPL-2.0 license?](https://opensource.stackexchange.com/questions/8831/pros-and-cons-of-using-mpl-2-0-license)
2. [Mozilla Public License 2.0](https://tldrlegal.com/license/mozilla-public-license-2.0-(mpl-2)) - especially "Disclose source" 

It's in our `devDependencies`, but still - probably it'd be worth to drop using any packages provided with this kind of license.

**Pull request merged**